### PR TITLE
migrate from make to invoke

### PR DESCRIPTION
**Pull request by flatnine**
- Migrate to use Python Invoke 
- Update README 
- Update version number

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.8
```
Comment (kthhrv) [inline requirements.txt:2]: should be 3.10?

File: tasks.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import subprocess
+import sys
+
+from invoke import task
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(levelname)s - %(message)s')
+
+APP_NAME = 'envars'
+PYTHON_VERSION = 3.10
```
Comment (kthhrv) [inline tasks.py:12]: should be loaded from the PYTHON_VERSION file?

File: tasks.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+@task
+def python_deps_upgrade(c):
+    """Update pip requirements"""
+    c.run('pip-compile --resolver=backtracking -U')
+    c.run('pip install -r requirements.txt')
+
+
+@task
+def python(c):
```
Comment (kthhrv) [inline tasks.py:53]: python and clean can be used from timeout-tools no need to include in every repo tasks.py

File: tasks.py (hunk 2 above)
Comment (flatnine) [reply tasks.py:12]: it actually falls back to this if a PYTHON_VERSION is not found.   I guess it should always be set

File: tasks.py (hunk 3 above)
Comment (flatnine) [reply tasks.py:53]: I don't use timeout-tools as I don't like the multi folder workflow.  

Do you just want me to delete python and clean?

**Review by kthhrv — APPROVED** [review]
LGTM

**Pull request merged**